### PR TITLE
Feature/add teams

### PR DESCRIPTION
**Pull request by mjholtkamp**

### CODE REVIEW

File: core/fields.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    Django's models are normally not aware of the request (by design),
+    so automatically saving the username of the person who made the change
+    to the object is not easy.
+
+    Using a global object can lead to all kinds of subtle bugs, so we
+    needed to find a way to pass the username in a different way; in this case
+    explicit.
+
+    :param model: The Django model to check for instances of AutoUsernameFields
```
[the remark this answers is not in the document]
Comment (mjholtkamp) [reply core/fields.py]: Agreed, I didn't notice, but my editor was still set to its default settings. I'll change that and this comment.

File: tests/settings.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    'django.contrib.contenttypes',
+
+    'core',
+
+    'encrypted_model_fields'
+]
+
+MIDDLEWARE = ()
+
+REQUESTS_CA_BUNDLE = "/etc/ssl/certs/ca-certificates.crt"
```
[the remark this answers is not in the document]
Comment (mjholtkamp) [reply tests/settings.py]: agreed! I'll change it, thanks!

File: core/models.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+from django.db import models
+
+from core.auditedmodel import AuditedModel
+
+
+class Team(AuditedModel):
+    public_identifier = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=100)
+    ldap_group = models.CharField(max_length=100)
```
[the remark this answers is not in the document]
Comment (mjholtkamp) [reply core/models.py]: Yes, it should :)

File: katka/admin.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+from katka.fields import username_on_model
+from katka.models import Team
+
+
+@admin.register(Team)
+class TeamAdmin(admin.ModelAdmin):
+    fields = ('name', 'group')
+
+    def save_model(self, request, obj, form, change):
+        with username_on_model(self.model, request.user.username):
```
Comment (CCardosoDev) [inline katka/admin.py]: Don't u want to unit test these lines?

Comment (mjholtkamp) [reply katka/admin.py]: Yes, I'll do that in a follow-up PR

**Pull request merged**